### PR TITLE
Adding simple pagination feature to the game browsing page

### LINKED ISSUE #286: Add pagination to game grid page
When we have a lot of games (like in the 50 or 500 games fixtures), they're displayed in an endless scroll on the game grid page. We want to add pagination to make it more manageable to browse games.

### PR DESCRIPTION
**Pull request by willcravitz**
We noticed that the game browse page caused you to scroll for way too long with the medium and large game fixtures. 
In issue #286:
1. We added Django pagination that splits the games up into pages of 20. 
2. We used the bootstrap pagination component to provide buttons for going to the first page, previous page number, display active page number, next page number, and last page. These appear disabled and cannot be clicked when appropriate.

### CODE REVIEW

File: src/templates/games/game_grid.html [hunk 1]
```diff
@@ -69,4 +69,37 @@ <h4 class="card-title">{{ game.name }}</h4>
       <a class="btn btn-outline-success" href="{% url 'game-create' %}">Create New Game</a>
     </div>
   {% endif %}
+  <div class="container mt-5 mb-3">
+    <nav>
+      <ul class="pagination pagination-lg justify-content-center">
```
Comment (rakrusch) [inline src/templates/games/game_grid.html:74]: I like the stylistic change to placing this in the center!

**Review by rakrusch — APPROVED** [review]
Pagination appears to be working properly. I have nothing else to add to this PR, everything looks really good! Great job!

**Review by majorsylvie — APPROVED** [review]
Looks quite nice! 

In the future you can consider adding more information to the pagination, like Last also saying what page number it is.

**Comment by willcravitz** [comment]
Closing comments: We added a pagination feature that splits up the games into pages of 20 games and provides a Bootstrap-styled pagination component at the bottom to navigate between these pages. This was achieved primary using Django's built in pagination feature. In future issues we will allow users to select the number of games per page (#297) and potentially update the pagination to display more page buttons, total number of pages, etc.

**Comment by majorsylvie** [comment]
Issue Score: Excellent

Comments:
Great work! Really appreciate you breaking down all the smaller components / specific features in the PR. Super readable and understandable.

Thank you for also including potential updates/extension! 

@rakrusch thank you for, while approving the PR, still actually commenting on it! This is better than a "LGTM" or "Looks Good" with no further elaboration or comment.

**Pull request merged**